### PR TITLE
added DB tests

### PR DESCRIPTION
**Pull request by DenisPitsul**

### CODE REVIEW

File: src/test/java/org/ssu/edu/teachua/ui/tasks/TasksPageTest.java [hunk 1]
```diff
@@ -1,6 +1,89 @@
...
+    @Test(description = "[Завдання] Verify that admin can create a task with valid data on 'Додайте завдання' page")
+    public void verifyThatAdminCanCreateTaskWithValidData() {
+        Calendar now = Calendar.getInstance();
+        String title = new HomePage(driver)
+                .getHeader()
+                .openAdminProfileMenu()
+                .openContentMenu()
+                .openChallengesMenu()
+                .clickTasks()
+                .openAddTaskPage()
```
Comment (janestadnytska) [inline src/test/java/org/ssu/edu/teachua/ui/tasks/TasksPageTest.java:67]: you can create '@BeforeMethod(description = "Precondition: open TasksPage _(or Open AddTaskPage)_")'

File: src/test/java/org/ssu/edu/teachua/ui/tasks/TasksPageTest.java [hunk 2]
```diff
@@ -1,6 +1,89 @@
...
+import java.util.List;
 
 public class TasksPageTest extends LoginWithAdminRunner {
+
+    private static final String NAME = StringGenerator.generateRandomString(20);
+    private static final String TITLE = StringGenerator.generateRandomString(50);
+    private static final String DESCRIPTION = StringGenerator.generateRandomString(200);
+    private static final String CHALLENGE = "Ukrainian";
+    private static final String PHOTO_PATH = "D:\\Projects\\teach_ua_739\\src\\test\\resources\\photos\\heart.png";
+
```
Comment (janestadnytska) [inline src/test/java/org/ssu/edu/teachua/ui/tasks/TasksPageTest.java:23]: you can move variables to DataProviderTask

**Comment by lhalam** [comment]
Conflicting files
src/test/java/org/ssu/edu/teachua/ui/tasks/TasksPageTest.java

**Comment by DenisPitsul** [comment]
Resolved merge conflict

**Review by lhalam — APPROVED** [review]

**Pull request merged**